c2/subprojects/utf8proc.wrap

```diff
@@ -0,0 +1,3 @@
+[wrap-git]
+url = https://github.com/JuliaStrings/utf8proc.git
+revision = 1cb28a66ca79a0845e99433fd1056257456cef8b
```

Comment:
You can use the meson port from `meson wrap install utf8proc` instead, no need to use cmake subprojects.